andela/AndelaEatsUI
# #158456041 Enable Admin to  view tapped users

#### What does this PR do?

- Enables Admin to retrieve all tapped users of a particular day

#### Description of Task to be completed?

- add TappedUsers component
- add Filter component
- add styles for the filter component
- modify App.js
- add getTappedUser actionTypes
- add tappedUsersReducer

#### How should this be manually tested?

- check out to this branch 
- run the application
- navigate to `/admin/users`
- click on **Tapped Users** tab
- By default, the tapped users for the last 6 days will be displayed on a graph
- Use the filter component to view tapped users for a specific date range

#### What are the relevant pivotal tracker stories?
- [#158456041](https://www.pivotaltracker.com/story/show/158456041)

#### Background Context
N/A

#### Screenshots (If Applicable)
![Jun-12-2019 11-57-34-graph](https://user-images.githubusercontent.com/9887151/59337495-63cb4a00-8d09-11e9-8657-51e15753ffa5.gif)
